packages/builder/src/components/common/NavItem.svelte

```diff
@@ -61,6 +62,7 @@
 
 <div
   class="nav-item"
+  class:hovering
```

Comment:
Why are we not using the default `hover`?

Reply:
Because this class is applied whenever the user is hovering over the corresponding component in the app preview, so it needs to be applied via js, I don't think there's any way we could do this with the `:hover` pseudo-class but I could be wrong.